src/crime_in_vancouver_eda.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Exploratory data analysis of Crime in Vancouver"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message=FALSE)
+library(tidyverse)
+library(knitr)
+library(ggridges)
+library(ggthemes)
+theme_set(theme_minimal())
+set.seed(2020)
+```
+
+# Summary of the data set
+
+```{r load data}
+crime_data <- read.csv("../data/crime.csv")
+crime_data |> 
+  drop_na() |> 
+  select(-X, -Y, -Latitude, -Longitude)
+```
+
+The data set used in this project comes from the Vancouver Open Data Catalogue. It includes 529.652 crimes reported between January 1, 2003 to July 13, 2017. The data set is also avialable on https://www.kaggle.com/wosaku/crime-in-vancouver
+
+```{r}
+# Type of crimes
+summary(crime_data)
+
+```
+```{r}
+# Count type of crimes
+type_crime <- crime_data |> 
+  group_by(TYPE) |> 
+  summarise(Count = n()) |> 
+  mutate(Percentage=paste0(round(Count/sum(Count)*100,2),"%")) |> 
+  arrange(desc(Count))
+type_crime
+
+```
+```{r}
+# Count Neighbourhood with most crimes
```

Comment:
I think this could be an interesting graph.

Reply:
Graph is added for the top neighbourhoods with most reported crimes. Decide not to plot all info otherwise the chart can be too intense.